packages/react-instantsearch-hooks-dom/src/widgets/__tests__/Hits.test.tsx

```diff
@@ -0,0 +1,172 @@
+import { render, waitFor } from '@testing-library/react';
+import React from 'react';
+import { InstantSearch } from 'react-instantsearch-hooks';
+
+import { createSearchClient } from '../../../../../test/mock';
+import {
+  createMultiSearchResponse,
+  createSingleSearchResponse,
+} from '../../../../../test/mock/createAPIResponse';
+import { Hits } from '../Hits';
+
+import type { InstantSearchProps } from 'react-instantsearch-hooks';
+
+const searchClient = createSearchClient();
+
+type SearchProviderProps = {
+  children: React.ReactNode;
+} & Partial<InstantSearchProps>;
+
+function SearchProvider({ children, ...props }: SearchProviderProps) {
+  return (
+    <InstantSearch searchClient={searchClient} indexName="indexName" {...props}>
+      {children}
+    </InstantSearch>
+  );
+}
+
+describe('Hits', () => {
+  test('renders with default props', async () => {
+    const { container } = render(
+      <SearchProvider>
+        <Hits />
+      </SearchProvider>
+    );
+
+    await waitFor(() => {
+      expect(container.querySelector('.ais-Hits')).toMatchInlineSnapshot(`
```

Comment:
Where does the wrapping `<div>` around `.ais-Hits` come from?

Reply:
Seems to be added by default by Testing Library. We could assert the full things though. This way we test we don't accidentally add any additional markup around the UI component in the widget.